code/hud/hudlock.cpp

```diff
@@ -719,7 +719,7 @@ void hud_do_lock_indicator(float frametime)
 	}
 }
 
-void hud_lock_acquire_current_target(object *target_objp, ship_subsys *target_subsys)
+void hud_lock_acquire_current_target(object *target_objp, ship_subsys **target_subsys)
```

Comment:
This will now modify the source target subsys, which is obviously the *intent* of the function but didn't happen before now.  Have the repercussions been checked?